ledger/ledger-utxo-observable/src/main/kotlin/net/corda/v5/ledger/utxo/observer/UtxoTokenPoolKey.kt

```diff
@@ -0,0 +1,22 @@
+package net.corda.v5.ledger.utxo.observer
+
+import net.corda.v5.crypto.SecureHash
+
+/**
+ * The [UtxoTokenPoolKey] defines a key for a pool of similar tokens.
```

Comment:
Should mention that we also segregate by notary automatically

Reply:
updated